source/azure_iot.c

```diff
@@ -36,7 +36,7 @@ static void prvAzureIoTLogListener( az_log_classification xClassification,
     /* In case logs are stripped out, suppress unused parameter error. */
     ( void ) xMessage;
 
-    AZLogInfo( ( "%.*s", az_span_size( xMessage ), az_span_ptr( xMessage ) ) );
```

Comment:
lets use the `int32_t` and `uint32_t` instead of `int` and `unsigned int` just to stick with the code style of this and FreeRTOS (if possible).

Reply:
Ok, I'll update the code.

Reply:
Done.